### PR TITLE
tup: allow overriding of CC and extra LDFLAGS in bootstrap

### PR DESCRIPTION
**Pull request by jmesmon**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ os=`uname -s`
 plat_cflags="`pkg-config fuse --cflags`"
 plat_ldflags="`pkg-config fuse --libs`"
 plat_files=""
-LDFLAGS="-lm"
-CC=gcc
+LDFLAGS="$LDFLAGS -lm"
+: ${CC:=gcc}
```
Comment (Qix-) [inline build.sh]: Could you explain this syntax?

Comment (jmesmon) [inline build.sh]: See [Shell Command Language (posix spec)](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html), `Assign Default Values`.

Comment (Qix-) [inline build.sh]: Gotcha, thanks!

**Comment by gittup** [comment]
Merged - thanks!

**Pull request closed without merging**